openserver/openserver.py

```diff
@@ -99,8 +99,13 @@ def DoGet(self, Gv):
             if value.isdigit():  # Checking if integer
                 value = int(value)
             elif '|' in value:  # Checking if | in string is returned
-                if any(x in Gv for x in (',', '[$]')):
-                    value = np.fromstring(value, sep="|")
+                if any(x in Gv for x in (',', '[$]', '@', ':', '_')):
```

Comment:
We are only testing DoGet with `[$]` and `[1:3]`. If `,`, `@` and `_` are valid options, they should be tested too, agreed?

Reply:
Removed '_' from wildcard check as it only retrieves the last item, never an array. 
Added two (2) testing functions, one for ',' and one for '@'. The latter relied on GAP model for testing, so gap_testfile added for this purpose.